tests/output-module-formats/subtests/es-modules/parcel/index.md

```diff
@@ -1,5 +1,13 @@
 ---
-result: pass
+result: fail
+issue:
+  - url: https://github.com/parcel-bundler/website/issues/631
```

Comment:
This issue is fixed :D

Reply:
Dunno if you saw https://github.com/GoogleChromeLabs/tooling.report/issues/373, but I'm having trouble with the check script here.